build_tools/buildkite/pipelines/postsubmit.yml

```diff
@@ -0,0 +1,37 @@
+# Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+agents:
+  queue: "orchestration"
+  security: "submitted"
+
+steps:
+  - label: ":hiking_boot: Bootstrapping postsubmit pipeline"
+    env:
+      PIPELINE_BOOTSTRAPPED: ${PIPELINE_BOOTSTRAPPED:-false}
+    commands: |
+      ./build_tools/buildkite/scripts/bootstrap_pipeline.sh \
+          build_tools/buildkite/pipelines/postsubmit.yml
```

Comment:
Seeing two of these may be confusing:

![image](https://user-images.githubusercontent.com/4010439/159952252-24dfd04b-38b7-40c1-b492-e295682e2c70.png)


(https://buildkite.com/iree/postsubmit/builds/40)

Can we differentiate them somehow?

Reply:
I can pull this all into buildkite if conditions so that the second bootstrap step gets skipped entirely, at the cost of more complex yaml. Thoughts? I can draft it up if you want to see what it would look like

Reply:
IMO let's avoid more complex yaml for now. That can be future cleanup once this is all working and other people start to notice the quirks. There may be larger usability issues at that point more worth the time.

Reply:
That was my thinking as well :slightly_smiling_face: I started trying to fiddle with it and then decided I should stop and actually get some real builds working